jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -303,7 +304,7 @@
                                     (send self :get-proximity l/r :left))
                                  (send self :interpolatingp))
                        (unix::usleep 1000))))
-                 (send self :cancel-move-hand)
+                 (send self :cancel-move-hand l/r)
                  (send self :cancel-angle-vector)
```

Comment:
If we have time, we should do:
- Fix `:cancel-angle-vector` and send PR to pr2eus
- Fix `(send self :cancel-angle-vector)` to `(send self :cancel-angle-vector :controller-type (send self :get-arm-controller arm))`